wand/image.py

```diff
@@ -2127,6 +2127,25 @@ def clear(self):
         """
         library.ClearMagickWand(self.wand)
 
+    def level(self, black, gamma, white, channel=None):
+        """Adjusts the levels of an image by scaling the colors falling between
+        specified white and black points to the full available quantum range.
+
```

Comment:
Missing Sphinx docblock. Eg :param: 

Change order to black, white, and gamma to match imagemagick' usage/example. Does't gamma have a default value?